src/Chapter2.hs

```diff
@@ -740,7 +774,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains el l = filter (\l1 -> elem el l1) l
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to other functions as well  🙂 
Here you can use it twice (on the function itself, and in the lambda function)
But your implementation is already great 💯